org.eclipse.jdt.core/formatter/org/eclipse/jdt/internal/formatter/CommentsPreparator.java

```diff
@@ -1250,9 +1386,14 @@ private boolean tokenizeMultilineComment(Token commentToken) {
 						i++;
 					position = i + 1;
 				} else if (!ScannerHelper.isWhitespace(c)) {
-					while (this.tm.charAt(i) == markerChar && lineBreaks > 0)
+					int markDownTaglen = 0;
+					while (this.tm.charAt(i) == markerChar && lineBreaks > 0) {
+						markDownTaglen++;
 						i++;
-					position = i;
+					}
+					position = (isMarkdown && markDownTaglen > MARKDOWN_TAG_LEN)
+							? i - (markDownTaglen - MARKDOWN_TAG_LEN)
+							: i;
```

Comment:
why not add condition `&& markerCharCount < 3` in the while loop above to avoid this complicated logic